package.json

```diff
@@ -49,7 +50,7 @@
     "tslint": "^4.0.0",
     "tslint-microsoft-contrib": "4.0.0",
     "typedoc": "^0.5.10",
-    "typescript": "^2.2.2"
+    "typescript": "2.3.0"
```

Comment:
Yea. I imported the linting config we use in other ts, which is all tslint 4. We need to update soon, but that involves updating tslint-contrib-microsoft too, which is ehh. For a followup PR!